modules/auth/auth_form.go

```diff
@@ -24,11 +24,13 @@ type AuthenticationForm struct {
 	AttributeName                 string
 	AttributeSurname              string
 	AttributeMail                 string
+	AttributeSSHPublicKey         string
 	AttributesInBind              bool
 	Filter                        string
 	AdminFilter                   string
 	IsActive                      bool
 	IsSyncEnabled                 bool
+	IsLdapPublicSSHKeySyncEnabled bool
```

Comment:
There is no need for this option as you can just check if `AttributeSSHPublicKey` is set